xilem/src/view/async_repeat.rs

```diff
@@ -65,12 +82,11 @@ where
 
     fn teardown(
         &self,
-        _: &mut Self::ViewState,
+        view_state: &mut Self::ViewState,
```

Comment:
I mean that's probably one of the nittiest nits:
```suggestion
        join_handle: &mut Self::ViewState,
```
as I think, then it's more obvious, what the view state is (as it's otherwise obvious via the type that it's the view state).

Reply:
Yeah. I don't like either of the options for different reasons, but I think this is better